anycloud/node/src/index.ts

```diff
@@ -1,7 +1,7 @@
-import { DataStore } from './datastore';
+import { datastore } from './datastore';
 
 export const PORT = 8088; // http://img2.wikia.nocookie.net/__cb20080604102014/starwars/images/3/3b/8t88.png
-export const DS = new DataStore();
+export const DS = datastore;
```

Comment:
So the `PORT` is all caps because it is an immutable constant, but this thing isn't, so I'm not sure about the allcaps namin here.